packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -972,6 +973,14 @@ export class WorkflowExecute {
 		return workflowIssues;
 	}
 
+	private getCustomOperation(node: INode, type: INodeType) {
+		const { resource, operation } = node.parameters ?? {};
+
+		if (typeof resource !== 'string' || typeof operation !== 'string') return;
```

Comment:
This also breaks if we'd have stricter TS types. The return type is `CustomOperationFn | undefined`, not `CustomOperationFn | void`

```suggestion
		if (typeof resource !== 'string' || typeof operation !== 'string') return undefined;
```